.drone.yml

```diff
@@ -1256,72 +1256,37 @@ depends_on:
 - default-windows-ltsc2022
 ---
 kind: pipeline
-name: docker-image-digests-linux-amd64
+name: docker-image-digests-linux
 
 platform:
   os: linux
   arch: amd64
 
 steps:
 - name: docker-image-digests
-  image: rancher/drone-docker-image-digests:v0.0.12
-  environment:
-    PLUGIN_GITHUB_REPOSITORY: "rancher/rancher"
-    PLUGIN_GITHUB_TOKEN:
-      from_secret: github_token
-    PLUGIN_GITHUB_TAG: "${DRONE_TAG}"
-    PLUGIN_INPUT_FILE: "rancher-images.txt"
-    PLUGIN_OUTPUT_FILE: "rancher-images-digests-linux-amd64.txt"
-    PLUGIN_REGISTRY: "docker.io"
+  image: rancher/dapper:v0.6.0
+  privileged: true
```

Comment:
If we can, it would be better not to use Dapper for this since we are trying to move away from having to use 
`privileged: true` in our pipelines.

Reply:
Can you share where this decision was made and what the new accepted way is? Or is it just as long as it is not privileged?

I think this is something that needs to be tackled outside this PR, as almost every step is following the same pattern and can be fixed if we have an agreed upon way how to move away from it.